pipelines/deployment.yml

```diff
@@ -4025,22 +4016,19 @@ disabled-jobs:
         inputs:
           - name: rm-survey-service-source
         outputs:
-          - name: rm-survey-service-build
-            path: rm-survey-service
+          - name: survey-service
         run:
           path: sh
           args:
           - -exc
           - |
-            mkdir -p /go/src/github.com/ONSdigital/
-            cp -r rm-survey-service-source /go/src/github.com/ONSdigital/rm-survey-service
-            make -C /go/src/github.com/ONSdigital/rm-survey-service
-            cp -r /go/src/github.com/ONSdigital/rm-survey-service/ .
+            cp -r rm-survey-service-source/* survey-service
+            make -C survey-service
   - put: cf-resource-prod
     params:
       current_app_name: rm-survey-service-concourse-prod
-      manifest: rm-survey-service-source/manifest.yml
-      path: rm-survey-service-build
+      manifest: survey-service/manifest.yml
```

Comment:
Not sure if we need these jobs (as in previous comment) but these manifests would be wrong I think as there are specific manifests for preprod and prod